keras_retinanet/models/retinanet.py

```diff
@@ -124,42 +124,65 @@ def default_regression_model(num_values, num_anchors, pyramid_feature_size=256,
     return keras.models.Model(inputs=inputs, outputs=outputs, name=name)
 
 
-def __create_pyramid_features(C3, C4, C5, feature_size=256):
+def __create_pyramid_features(C3, C4, C5, pyramid_levels, C2=None, feature_size=256):
```

Comment:
Shouldn't this just take the dict now?

Reply:
Good point. Everything now works with dicts end to end.